cmake/Dependencies.cmake

```diff
@@ -440,7 +457,6 @@ if(USE_XNNPACK AND NOT USE_SYSTEM_XNNPACK)
   endif()
 
   if(NOT TARGET XNNPACK)
-    set(XNNPACK_CUSTOM_THREADPOOL ON CACHE BOOL "")
```

Comment:
There is no such thing as XNNPACK_CUSTOM_THREADPOOL.  This flag is non existent.